test/support/factory.ex

```diff
@@ -23,7 +24,8 @@ defmodule Oli.Factory do
       name: sequence("User name"),
       given_name: "User given name",
       family_name: "User family name",
-      sub: "#{sequence("usersub")}"
+      sub: "#{sequence("usersub")}",
+      author: insert(:author)
```

Comment:
What does an author of an author mean?

Reply:
Didn't get your question. This is the user's linked author